config/jenkins/configuration/clouds.yml

```diff
@@ -204,6 +245,47 @@ jenkins:
         installGit: false
         installMaven: false
         javaPath: "java"
+        labels: "ACC-Windows-2016"
```

Comment:
The labels for ws2019-SGX-DCAP and ws2016-nonSGX are both "ACC-Windows-2016", which means Jenkins is attempting to spin up either one when that label is needed to build a job. Is this the intended behaviour?